nilearn/image/image.py

```diff
@@ -536,14 +537,19 @@ def iter_img(imgs):
 
     Parameters
     ----------
-    imgs: 4D Niimg-like object
+    imgs: 3D or 4D Niimg-like object
         See http://nilearn.github.io/manipulating_visualizing/manipulating_images.html#niimg.
 
     Returns
     -------
     output: iterator of 3D nibabel.Nifti1Image
     """
-    return check_niimg_4d(imgs, return_iterator=True)
+
+    try:
+        return check_niimg_4d(imgs, return_iterator=True)
+    except DimensionError:
+        # Wrap a 3D image in a dummy list.
+        return check_niimg_4d([imgs], return_iterator=True)
```

Comment:
Indeed, I no longer use `iter_img` in `plot_ica_neurovault.py` after previous changes, so this is no longer needed. Will pull this and close the PR.